operations/operations/zuul-ci-cd-quickstart-user-guide.md

```diff
@@ -0,0 +1,203 @@
+# Zuul users guide
+
+## Prerequisites
+
+1. Repository is known by your zuul instance
```

Comment:
I would prefer the documentation to be SCS specific and refer to the SCS Zuul instance (ideally with a link).
Also, how do I find out whether the repository is known to the instance? Who could tell me about that and (if necessary) add the repo?